### ISSUE TITLE
Exclude relative path

### ISSUE THREAD

**Issue by MJAS1** [rizsotto/Bear#387]
Accroding to [this](https://github.com/rizsotto/Bear/blob/master/source/citnames/man/citnames.1.md#config-file) and my own testing, it seems that I can use `paths_to_exclude` to only exclude absolute paths. If I remember correctly, I was able to use an earlier version of Bear with `--exclude` flag to exclude relative paths. Is there really no way to currently achieve the same thing? I am trying to setup a CI/CD step that will not work with absolute paths since it might not be known beforehand.

**Comment by rizsotto** [comment]
Hey @MJAS1 , thanks for the report. 

Yes, you are right the 2.4 version had this code to fix the relative path for the include/exclude directory list.

```python
def make_absolute(directory):
    # type: (str) -> str  
    """ Makes a path like object absolute (to the project root). """
    if os.path.isabs(directory):
        return directory
    else:
        return os.path.normpath(os.path.join(os.getcwd(), directory))
```
The current implementation using the include/exclude directories without this check. (`source/citnames/source/Output.cc:44` the `ContentFilter` class needs to be changed for that.)

If I understand correctly, you would prefer to use the current directory as a base to the relative paths. Is this correct?

**Comment by MJAS1** [comment]
Yes, you are correct. I was trying to use bear in a GitLab CI/CD pipeline step. The problem was that, as far as I know, there is no way to know beforehand the full path as the pipeline is ran inside a Docker container that contains a folder named after the username of the person initiating the pipeline. Finally, I ended up going a different route but anyways I think it would be nice if you could use a relative path.

**Comment by rizsotto** [comment]
I think this is fixed now on latest `master` branch. Will be released as `3.0.13` soon.

@MJAS1 could you verify that it works for you?

**Comment by MJAS1** [comment]
I just tested this and it seems to work. Thanks for the great work!
